packages/size-limit/read-pkg-up.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path')
+const { promisify } = require('util')
+const fs = require('fs')
+
+const readFileAsync = promisify(fs.readFile)
```

Comment:
Use `fs/promises` instead

Reply:
I see that configured version of node is `10^ || ...`, and `require('fs/promises')` are supported from version 12, and `require('fs').promises` are supported from version 11

Reply:
Got it 👍